src/main/java/core/basesyntax/service/impl/operation/PurchaseOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.service.OperationHandler;
+
+public class PurchaseOperation implements OperationHandler {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.

Reply:
--